web/packages/teleport/src/Integrations/status/AwsOidc/Tasks/Tasks.tsx

```diff
@@ -0,0 +1,162 @@
+    },
+    clusterId: '',
+    params: {},
+  });
+
+  useEffect(() => {
+    serverSidePagination.fetch();
+  }, [integration]);
+
+  if (integrationAttempt.status === 'processing') {
+    return <Indicator />;
+  }
+
+  if (serverSidePagination.attempt.status === 'processing') {
+    return <Indicator />;
+  }
+
+  if (integrationAttempt.status === 'error') {
+    return <Danger>{integrationAttempt.statusText}</Danger>;
+  }
+
+  if (serverSidePagination.attempt.status === 'failed') {
+    return <Danger>{serverSidePagination.attempt.statusText}</Danger>;
+  }
+
+  function close(resolved: boolean) {
+    if (resolved) {
+      serverSidePagination.modifyFetchedData(p => ({
+        ...p,
+        agents: p.agents.filter(t => t.name !== showTask.name),
+      }));
+    }
+    setShowTask(undefined);
+  }
+
+  return (
+    <Flex>
+      <Flex
+        flexDirection="column"
+        css={`
+          flex-grow: 1;
+        `}
+      >
+        {integration && (
+          <AwsOidcHeader integration={integration} tasks={true} />
+        )}
+        <FeatureBox
+          css={{ maxWidth: '1400px', paddingTop: '16px', gap: '30px' }}
+        >
+          {integration && (
+            <AwsOidcTitle integration={integration} tasks={true} />
+          )}
+          <Table<UserTask>
+            data={serverSidePagination.fetchedData?.agents || []}
+            row={{
+              onClick: row => {
+                if (showTask === undefined) {
+                  setShowTask(row);
+                }
+              },
+              getStyle: () => {
+                if (showTask === undefined) {
+                  return { cursor: 'pointer' };
+                }
+              },
+            }}
+            columns={[
+              {
+                key: 'taskType',
+                headerText: 'Type',
+                render: (item: UserTask) => (
+                  <Cell>{getResourceType(item.taskType).toUpperCase()}</Cell>
+                ),
+              },
+              {
+                key: 'issueType',
+                headerText: 'Issue Details',
+              },
+              {
+                key: 'lastStateChange',
+                headerText: 'Timestamp (UTC)',
+                render: (item: UserTask) => (
+                  <Cell>{new Date(item.lastStateChange).toISOString()}</Cell>
+                ),
+              },
+              {
+                altKey: 'action',
+                headerText: 'Actions',
+                render: (item: UserTask) => (
+                  <Cell>
+                    <ButtonBorder
+                      onClick={() => setShowTask(item)}
+                      disabled={showTask != undefined}
+                      size="small"
+                    >
+                      View
+                    </ButtonBorder>
+                  </Cell>
+                ),
+              },
+            ]}
+            emptyText={`No pending tasks`}
+            fetching={{
```

Comment:
You also need to add set
```ts
pagination={{ pageSize: serverSidePagination.pageSize }}
```
otherwise the pagination buttons won't show up 💀 

You can also set the buttons to be shown at the bottom of the page to avoid a large gap between the page title and the table.